cmd/buck/cli/init.go

```diff
@@ -73,8 +73,10 @@ Use the '--hard' flag to discard all local changes.
 			cmd.Fatal(errors.New("--cid cannot be used with an existing bucket"))
 		}
 
-		unfreeze, err := c.Flags().GetBool("unfreeze")
-		cmd.ErrCheck(err)
+		// (jsign): re-enable when this feature is usable in mainnet.
+		//unfreeze, err := c.Flags().GetBool("unfreeze")
+		//cmd.ErrCheck(err)
+		var unfreeze bool
```

Comment:
I guess we need to include some test to compile all the project to catch these things.

Reply:
We have those tests (added a month ago or so). It did compile, but `init` would throw a runtime error.

Reply:
Ah, I read this diff wrong. Too early in the morning.